camelot/view/action_steps/select_file.py

```diff
@@ -68,13 +68,10 @@ class SelectFile( ActionStep, DataclassSerializable ):
     """
 
     file_name_filter: str = ''
-    single: bool = field(init=False)
+    single: bool = True
 
     caption = _('Open')
 
-    def __post_init__(self):
-        self.single = True
```

Comment:
Dit is nog het werk van de studenten denk ik. Deze optie w niet gebruikt denk ik maar ik zie niet meteen wat hier de bedoeling was...

Reply:
bedoeling was denk ik om die single parameter uit de constructor te houden, en hem toch te initialiseren

of dat nu erg is dat ie wel in de constructor zit, dat is iets anders ....